src/main/java/duke/command/AddCommand.java

```diff
@@ -5,11 +5,21 @@
 import duke.util.Ui;
 import duke.task.Task;
 
+/**
+ * Create an AddCommand. It adds tasks to user's list.
```

Comment:
Comment should start with "Creates" and not "Create". The same applies to the comments for the rest of the methods.